portal/views/patients.py

```diff
@@ -95,8 +95,25 @@ def patients_root():
                      UserIntervention.intervention_id.in_(ui_list)))
         patients = patients.union(ui_patients)
 
+    #get assessment status only if it is needed as specified by config
+    patient_list_addl_fields = current_app.config.get('PATIENT_LIST_ADDL_FIELDS')
+    if 'status' in patient_list_addl_fields:
+        patient_list = []
+        for patient in patients:
+            assessment_status = AssessmentStatus(user=patient)
+            try:
+                patient.assessment_status = (
+                    assessment_status.overall_status if assessment_status else
+                    None)
+            except ValueError:
+                patient.assessment_status = None
+                current_app.logger.debug("Error retrieving assessment status for patient {}".format(str(patient.id)))
+
+            patient_list.append(patient)
+        patients = patient_list
```

Comment:
reassigning to the same variable name `patients` isn't a great idea.  just modify how it's passed to the template below a la `pateints_list=patients_list`